src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  margin: 0;
+  padding: 0;
```

Comment:
The `margin` and `padding` properties are redundant here since they are already set to 0 by default for the `html` element.